### DISCUSSION TITLE
Changing card background color dynamically

### DISCUSSION THREAD

**Discussion by hanymorcos** [tailwindlabs/tailwindcss#8664] [Help]
Hi,

I'm trying to change bg color dynamically on a card and nothing is happening. I see the color changes in the inspect using chrome, but the actual color is not changing.

Thanks,

Hany

import React from 'react';

export const AwesomeBlob = ({
  title,
  content,
  id,
}) => {
  const colors = ['orange', 'amber','slate','lime','cyan','violet']
  const random = Math.floor(Math.random() * colors.length);
  const className = "shadow  max-w-md bg-" + colors[random] + "-300 rounded"

  return (
    <div key={id} className={className}>
      <div className="p-5 flex flex-col space-y-2">
        <p className="text-lg font-medium">{title}</p>
        <p className="text-gray-600">{content}</p>
      </div>
    </div>
  );
};

**Comment by timohoelzer** [comment]
Hi, 

seams the problem is that your dynamic classes could not be detected by tailwind because
they are nowhere written in total e.g. **bg-orange-300**
So the classes of your choice are never rendered to your CSS.

try this instead
`const colors = ['bg-orange-300', 'bg-amber-300','bg-slate-300','bg-lime-300','bg-cyan-300','bg-violet-300']`

`const className = "shadow max-w-md " + colors[random] + " rounded"`
